internal/xds/translator/route.go

```diff
@@ -26,6 +26,13 @@ func buildXdsRoute(httpRoute *ir.HTTPRoute) (*route.Route, error) {
 		ret.RequestHeadersToRemove = httpRoute.RemoveRequestHeaders
 	}
 
+	if len(httpRoute.AddResponseHeaders) > 0 {
+		ret.ResponseHeadersToAdd = buildXdsAddedResponseHeaders(httpRoute.AddResponseHeaders)
```

Comment:
can we make the method for Request and Response common to reduce code duplication ?